### DISCUSSION TITLE
중급 프로젝트 2주차 회고

### DISCUSSION THREAD

**Discussion by Crack-Egg** [Party-Time-Job/Party-Time-Job#68] [회고록]
# Keep

현재 우리팀이 잘 하고 있어서 계속 유지하면 좋을 것 같은 부분을 작성해주세요.
이시간을 빌려 한주간 프로젝트에서 잘 된 점들을 서로 마구 칭찬해주는 시간을 가져도 좋겠네요.

## Problem

현재 우리팀이 가지고 있는 문제가 있다면 작성해주세요.
문제의 발견은 곧 우리 팀이 더 성장할 수 있다는 의미이니, 문제 발견을 두려워하지말아요:)

## Try

Problem에 적은 것을 해결하기 위해 우리 팀이 다음에 시도해볼만한 것 혹은 Keep 중에서도 더 개선하거나 시도해보고 싶은 것을 작성해보세요. 다음주에는 Try에 적은 것을 바탕으로 더 좋은 협업을 해요 :)

**Comment by yunsunglee2** [comment]
**Keep**
- PR 요청시 서로 코드리뷰 해주며 팀원이 어떻게 작업했는지 확인 할 수 있는 것.
- github의 기능으로 프로젝트의 전반적인 진행상황을 확인하는 것.
**Problem**
- 프로젝트의 효율적인 진행을 위해서 컨벤션 또는 쓰기로 한 기능을 제외하고 나중에 리팩토링으로 미루어 놓는 것
**Try**
- 익숙치 않은 기능과 컨벤션이라도 최대한 맞춰가려고 하며 프로젝트를 진행해 본다

**Comment by Crack-Egg** [comment]
## Keep

- 각자의 진행상황에 맞게 스케줄을 잘 관리했고, 폴더 구조에 맞게 작업을 진행하는 것이 좋았습니다. 또, 서로 잘 모르는 부분에 있어서 항상 자세히 팀원들끼리 알려주고, 어려운 부분은 같이 해결하려는 모습이 좋았습니다.

## Problem

- 개인적으로 tailwind 사용법에 많이 익숙하지 않아, 스타일링 작업을 뒤로 미루었는데, 여러 스타일링을 한번에 리팩토링해야 하다보니 더 시간도 많이 걸리는 것 같습니다.

## Try

- 기본적인 전체 스타일을 우선적으로 적용시키고, 어려운 부분은 주석을 남겨놓고, 나중에 수정해야 하는 부분을 파악해 놓도록 하겠습니다.

**Comment by DanteSnow** [comment]
# Keep

각자 맡은 역할에 대해서 최선을 다해 임하는 모습이 좋았습니다.
다른 팀원의 문제도 자신의 문제처럼 같이 들여다 보는 모습이 좋았습니다.

## Problem

기능을 구현하기 위한 선행 학습이 부족해서 정석적인 방법이 아닌 돌아가는 방법을 택했습니다. (Next14, 쿠키, 인증 등)

## Try

이번 기회에 확실하게 학습한다고 생각하며 최선을 다해 임할 것입니다.

**Comment by siyeol97** [comment]
## Keep
- 맡은 기능을 열심히 구현하는 것이 좋았습니다.
- PR을 빠르게 확인해주셔서 좋았습니다.

## Problem
- 구현해야 할 기능을 상세히 적어놓고 차례대로 구현하지 않고, 대략적으로 구현을 해놓고 추가해야 할 기능을 작업하느라 일정이 뒤로 미뤄졌습니다.

## Try
- 같은 실수를 반복하지 않게끔 R&R과 각 기능들을 상세히 적어놓겠습니다
